regression/goto-cc-instrument-cbmc/identifier_names_illegal_characters/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+test.c
+
```

Comment:
:no_entry_sign: Positive regexes for how the function declarations are printed and negative regexes for what they shouldn't be?

Reply:
Could they be better left in the comment section? :thinking: 

In this particular case, for this pipeline (goto-cc -> goto-instrument -> cbmc), the end result if the function declarations are wrong is that `cbmc` results in a parsing error.

To implement your change, I imagine I have to remove cbmc from there, and do regex pattern matching on the `--dump-c` output of `goto-instrument`, which I think is less ideal for our usecase.

Reply:
Good point - this is fine